### PR TITLE
virttest.utils_test.libvirt: Create volume by xml file.

### PR DESCRIPTION
**Pull request by rbian**
Add a function in PoolVolumeTest to create volume by xml file.

### CODE REVIEW

File: virttest/utils_test/libvirt.py [hunk 1]
```diff
@@ -1044,6 +1045,16 @@ def pre_vol(self, vol_name, vol_format, capacity, allocation, pool_name):
         if not pv.volume_exists(vol_name):
             raise exceptions.TestFail("Can't find volume: %s" % vol_name)
 
+    def pre_vol_xml(self, pool_name, **vol_params):
```
Comment (will-Do) [inline virttest/utils_test/libvirt.py]: This function create a volume, but from the name, it more like create a XML file, so I suggest you can change the name to 'pre_vol_by_xml'

**Comment by rbian** [comment]
Thanks @will-Do , Fixed.

**Comment by PandaWei** [comment]
ACK. I'm OK for this PR.

Just curious, why give check_exit_status the second paramter false?
It's default is false.

**Comment by will-Do** [comment]
ACK

**Pull request merged**